alternator/executor.cc

```diff
@@ -3262,7 +3263,14 @@ future<executor::request_return_type> executor::get_item(client_state& client_st
     const rjson::value* expression_attribute_names = rjson::find(request, "ExpressionAttributeNames");
     verify_all_are_used(expression_attribute_names, used_attribute_names, "ExpressionAttributeNames", "GetItem");
 
-    return _proxy.query(schema, std::move(command), std::move(partition_ranges), cl,
+    if (!co_await client_state.check_has_permission(auth::command_desc(
+            auth::permission::SELECT,
+            auth::make_data_resource(schema->ks_name(), schema->cf_name())))) {
+        co_return api_error::access_denied(format(
+            "SELECT permissions denied on {}.{} by RBAC", schema->ks_name(), schema->cf_name()));
```

Comment:
I don't know if it's a copy-paste, but I think it could be a more proper English, for example,
"permission is denied on" or better yet "SELECT access is denied to "

Reply:
I don't remember how I wrote the first message of this type, but afterwards I did copy them so all of them got the same not-great wording. I'll create a patch to improve the messages.